maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/docker-compose.yml

```diff
@@ -8,15 +8,19 @@ services:
     image: gcr.io/kalix-public/kalix-runtime:1.1.32
     container_name: ]]#${artifactId}#[[
     ports:
-      - "9000:9000"
+      - "${ADVERTISED_HTTP_PORT}:9000"
     extra_hosts:
       - "host.docker.internal:host-gateway"
     environment:
       JAVA_TOOL_OPTIONS: >
       # jvm -D properties can be added under this environment map (note: remove this comment when adding properties)
 
+      # those variables are defined in the .env file
+      # when running multiple services on your local machine, make sure that 
+      # the ports are unique by editing the corresponding .env files
+      ADVERTISED_HTTP_PORT: ${ADVERTISED_HTTP_PORT}
```

Comment:
It's a pity that we need to repeat the env vars to pass it to the JVM. 

Also, I'm adding this text here because the service port is also defined here and it's better to get them all together. But `ADVERTISED_HTTP_PORT` shows up for the first time in the ports mapping above. 

Anyway, that might be fine an obvious enough for users.